updater/lib/dependabot/setup.rb

```diff
@@ -39,7 +41,8 @@
     swift
   )}x
 
-  config.processors += [ExceptionSanitizer]
+  config.before_send = ->(event, hint) { Dependabot::Sentry::Processor.process_chain(event, hint) }
```

Comment:
`config.processors` has been removed in favour of a lambda `before_send`. I've replicated the behaviour by using a reducer to call processors in order.

I had to migrate `raven_context` to a custom processor as well, because `raven_context` is no longer attached to events by default.